tools/gulp/tasks/lint.ts

```diff
@@ -46,6 +54,56 @@ task('madge', ['material:clean-build'], () => {
   });
 });
 
+task('ownerslint', () => {
+  let errors = 0;
+
+  let ownedPaths = readFileSync(ownersFilePath, 'utf8').split('\n')
+      // Trim lines.
+      .map(line => line.trim())
+      // Remove empty lines and comments.
+      .filter(line => line && !line.startsWith('#'))
+      // Split off just the path glob and turn it into a regex.
+      .map(line => new Minimatch(line.split(/\s+/)[0], {dot: true, matchBase: true}))
+      // Add in the paths we're ignoring.
+      .concat(ignoreOwners.map(path => new Minimatch(path, {dot: true, matchBase: true})));
+
+  for (let paths = getChildPaths('.'); paths.length;) {
+    paths = Array.prototype.concat(...paths
+        // Remove paths that match an owned path.
+        .filter(path => !ownedPaths.reduce(
+            (isOwned, ownedPath) => isOwned || isOwnedBy(path, ownedPath), false))
+        // Report an error for any files that didn't match any owned paths.
+        .filter(path => {
+          if (statSync(path).isFile()) {
+            console.error(red(`No code owner found for "${path}".`));
```

Comment:
If you want to make this look like a Gulp task out, you can use the `gulp-util`.